--- origin/dev:AnalogBoard_Dll/AnalogBoard_Dll.h
+++ temp:AnalogBoard_Dll/AnalogBoard_Dll.h
@@ -5,8 +5,8 @@
 * File name		:	AnalogBoard_Dll.h
 * File summary	:	USB_Lib_Info class define
 *******************************************************************************/
-#ifndef _ANALOGBOARD_DLL_H_
-#define _ANALOGBOARD_DLL_H_
+#ifndef _SYSMEX_ANALOGBOARD_DLL_H_
+#define _SYSMEX_ANALOGBOARD_DLL_H_
 
 #pragma once
 
@@ -71,4 +71,4 @@
 }
 #endif
 
-#endif // !_ANALOGBOARD_DLL_H_
+#endif // !_SYSMEX_ANALOGBOARD_DLL_H_
