--- origin/dev:AnalogBoard_TestApp/resource.h
+++ temp:AnalogBoard_TestApp/resource.h
@@ -6,7 +6,7 @@
 #define IDM_ABOUTBOX                    0x0010
 #define IDD_ABOUTBOX                    100
 #define IDS_ABOUTBOX                    101
-#define IDD_ANALOGBOARD_TESTAPP_DIALOG 102
+#define IDD_SYSMEX_ANALOGBOARD_TESTAPP_DIALOG 102
 #define IDR_MAINFRAME                   128
 #define IDD_DIALOG1_MAIN                130
 #define IDD_DIALOG2_DEBUG               132
