--- origin/dev:AnalogBoard_Dll/AnalogBoard_Dll.cpp
+++ temp:AnalogBoard_Dll/AnalogBoard_Dll.cpp
@@ -32,7 +32,7 @@
 const char* g_DLLVersion = "1.0.0";
 
 /* {A123DFB8-6F1E-49F4-93F4-C4E4B48FD5BD} */
-static GUID USBDRV_GUID = { 0xA123DFB8, 0x6F1E, 0x49F4, 0x93, 0xF4, 0xC4, 0xE4, 0xB4, 0x8F, 0xD5, 0xBD };
+static GUID SYSMEXUSBDRV_GUID = { 0xA123DFB8, 0x6F1E, 0x49F4, 0x93, 0xF4, 0xC4, 0xE4, 0xB4, 0x8F, 0xD5, 0xBD };
 
 /* The USB Device Class */
 static CCyUSBDevice* m_pUSBDevice;
